packages/extras/src/lib/components/AnimatedSpriteMaterial/AnimatedSpriteMaterial.svelte

```diff
@@ -0,0 +1,269 @@
+<script lang='ts'>
+  import * as THREE from 'three'
```

Comment:
This is probably not a problem, but don't we usually use named imports instead of a namespace THREE?

Reply:
Oh, you're right! Missed that one 👍